codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'package:codesign/codesign.dart';
+import 'package:file/file.dart';
+import 'package:process/process.dart';
+
+/// Interface for classes that interact with files nested inside of [RemoteZip]s.
+abstract class FileVisitor {
```

Comment:
Maybe you have this abstract for some future usage, but as-written I think you should just use `FileCodesignVisitor`, the interface isn't really necessary.

Reply:
I don't really have a preference, will let Chris chime in.

Reply:
Revisiting this. What's the value having an abstract class is giving us? Do we have other versions planned?